### ISSUE TITLE
CLI - Remove dependency on NetMQ and centralize microservice-related listen in `project ps`

### ISSUE THREAD

**Issue by PedroRauizBeamable** [beamable/BeamableProduct#3606]
We have 3 sources of microservice state:
- Remote/Local Manifest
- The NetMQ Beacon (or rather a localhost broadcast made by any service running locally)
- The notification Beamo now emits whenever a service registers itself.

The `CheckStatusCommand` (`project ps -w`) is supposed to keep watch for changes in the topology of running microservices locally (and now, in the realm) and provide a unified interface about the state of each microservice in a realm. We can then build engine integrations on top of this interface.

In other words, it'd be nice to be able to have this command be the entire source of truth about microservice state inside engine integrations. So... that what this ticket is supposed to do.

As a side note, NetMQ has proven to be a bit annoying with hard-to-debug issues a few times. Given that we only need a simple UDP broadcast within the local machine, we should just write our own little socket-based broadcast/listen pattern and remove this depedency.
